analytics-api/requirements.txt

```diff
@@ -1,17 +1,17 @@
 Flask-Caching==2.1.0
-Flask-Cors==6.0.0
```

Comment:
Interesting that the analytics-api was running on a much newer version than everything else.